connections/sim/hw/sensors/voltage_sensor_sim.py

```diff
@@ -0,0 +1,37 @@
+from connections.sim.hw.sensors.sensor import *
+
+
+class VoltageSensor(Sensor):
+    """
+    Simulates the hardware used for measuring battery voltage.
+    """
+
+    # 920mV is the value that gets converted to 11.6V in battery.cpp in FLARE 21899292dc39015570f795ef9e607081aab57e3e
```

Comment:
The value we return in `read()` gets spit out in battery.cpp from the call to `Hal::analogRead(m_batterySensorPin)`. Hal::analogRead is a wrapper around the [arduino analog read](https://www.arduino.cc/reference/en/language/functions/analog-io/analogread/). This returns raw ADC level as an integer between 0 and 2^10-1, not mV. You can see that battery.cpp does the conversion from ADC level to pin voltage first, and then pin voltage to battery voltage.

Some of the variables and comments in this file refer to this value as a "voltage" which is misleading. Others refer to it as a "voltage level" which is a bit better but also ambiguous. I think "ADC level" is clearest, but you can use whatever. Even "voltage level" is okay as long as we have comments that clearly explain what it actually is.